cinema/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
Consider registering your models with the Django admin site by importing them and using `admin.site.register(ModelName)`.